code401Challenges/src/test/java/code401Challenges/hashtable/LeftJoinTest.java

```diff
@@ -1,7 +1,46 @@
 package code401Challenges.hashtable;
 
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
 import static org.junit.Assert.*;
 
 public class LeftJoinTest {
+    Hashtable hashtable1;
+    Hashtable hashtable2;
+
+    @Before
+    public void setUp () {
+
+        hashtable1 = new Hashtable();
+        hashtable1.add("earth", "terra");
+        hashtable1.add("happy", "joyful");
+        hashtable1.add("active", "energetic");
```

Comment:
Since the hashtable will add the key/value pair into whatever index of the array the key hashes to, you can't assume that they will be in the above order when you make your `expected` ArrayList in your tests.